Solutions/McAfee Network Security Platform/Data Connectors/McAfeeNSP.json

```diff
@@ -0,0 +1,128 @@
+{
+    "id": "McAfeeNSP",
+    "title": "McAfee Network Security Platform",
+    "publisher": "McAfee",
+    "descriptionMarkdown": "The [McAfee® Network Security Platform](https://www.mcafee.com/enterprise/en-us/products/network-security-platform.html) data connector provides the capability to ingest [McAfee® Network Security Platform events](https://docs.mcafee.com/bundle/network-security-platform-10.1.x-integration-guide-unmanaged/page/GUID-8C706BE9-6AC9-4641-8A53-8910B51207D8.html) into Azure Sentinel. Refer to [McAfee® Network Security Platform](https://docs.mcafee.com/bundle/network-security-platform-10.1.x-integration-guide-unmanaged/page/GUID-F7D281EC-1CC9-4962-A7A3-5A9D9584670E.html) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**McAfeeNSPEvent**](https://aka.ms/sentinel-mcafeensp-parser) which is deployed with the Azure Sentinel Solution.",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "McAfeeNSP",
+        "baseQuery": "McAfeeNSPEvent"
+    }],
+    "sampleQueries": [{
+        "description": "Top 10 Sources",
+        "query": "McAfeeNSPEvent\n | summarize count() by tostring(DvcHostname)\n | top 10 by count_"
+    }],
+    "dataTypes": [{
+        "name": "Syslog(McAfeeNSPEvent)",
+        "lastDataReceivedQuery": "McAfeeNSPEvent\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+    }],
+    "connectivityCriterias": [{
+        "type": "IsConnectedQuery",
+        "value": [
+            "McAfeeNSPEvent\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+        ]
+    }],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [{
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
```

Comment:
SharedKeys permission is not required in case of Syslog connector, so please remove it.
Please refer the syslog template [here](https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_Syslog_template.json)

Reply:
updated